cmis_web_proxy/static/src/js/form_widget.js

```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------
+ + * Odoo web_cmis_viewer
+ + * Author  Laurent Mignon 2016 Acsone SA/NV
+ + * License in __openerp__.py at root level of the module
+ + *---------------------------------------------------------
+ +*/
+
+ odoo.define('cmis_web_alf.form_widgets', function( require) {
+"use strict";
+
+var core = require('web.core');
+var form_widgets = require('cmis_web.form_widgets');
+var Model = require('web.Model');
+var $ = require('$');
```

Comment:
@max3903 This line is not working in Odoo10.  `+var $ = require('$');` It must be removed.